nomos-cli/src/cmds/executor.rs

```diff
@@ -80,3 +75,17 @@ impl Disseminate {
         Ok(())
     }
 }
+
+#[tokio::main]
```

Comment:
This is a nice trick! Didn't know we could do that.

Reply:
Learned it from @zeegomo 💪